pkgs/development/libraries/lpcnetfreedv/default.nix

```diff
@@ -1,30 +1,19 @@
 { lib, stdenv, fetchFromGitHub, fetchurl, cmake, codec2 }:
 
-let
-  dataVersion = "191005_v1.0";
-  data = fetchurl {
-    url = "http://rowetel.com/downloads/deep/lpcnet_${dataVersion}.tgz";
-    sha256 = "1j1695hm2pg6ri611f9kr3spm4yxvpikws55z9zxizai8y94152h";
-  };
-in stdenv.mkDerivation rec {
+stdenv.mkDerivation rec {
   pname = "lpcnetfreedv";
-  version = "unstable-2021-06-29";
+  version = "0.3";
```

Comment:
Using version `unstable-2022-08-22` / rev `67a6eb74d0c07faddcdce199856862cc45779d25` (current master) should solve the circular dependency issue (cf. <https://github.com/NixOS/nixpkgs/compare/master...illdefined:nixpkgs:freedv-1.8.4>)

Reply:
Done. Thanks!